apps/meteor/client/sidebar/RoomList/SideBarItemTemplateWithData.js

```diff
@@ -136,7 +136,11 @@ const propsAreEqual = (prevProps, nextProps) => {
 	if (prevProps.room._updatedAt?.toISOString() !== nextProps.room._updatedAt?.toISOString()) {
 		return false;
 	}
-	if (prevProps.room.lastMessage?._updatedAt?.toISOString() !== nextProps.room.lastMessage?._updatedAt?.toISOString()) {
+	if (
+		typeof prevProps.lastMessage?._updatedAt === 'object' &&
```

Comment:
If _updatedAt happens to be null, this will fail 👀

Reply:
Good catch!! Modified it to handle this case